wire-library/wire-schema-tests/build.gradle.kts

```diff
@@ -2,26 +2,49 @@ import com.vanniktech.maven.publish.JavadocJar.Dokka
 import com.vanniktech.maven.publish.KotlinJvm
 import com.vanniktech.maven.publish.MavenPublishBaseExtension
 
-// TODO(Benoit) this module can be multiplatform.
-
 plugins {
-  kotlin("jvm")
+  kotlin("multiplatform")
   id("org.jetbrains.dokka")
   id("com.vanniktech.maven.publish.base")
 }
 
-dependencies {
-  api(deps.junit)
-  api(project(":wire-compiler"))
-  api(project(":wire-schema"))
-  implementation(project(":wire-kotlin-generator"))
-  implementation(project(":wire-java-generator"))
-  implementation(project(":wire-swift-generator"))
-  implementation(deps.okio.core)
-  implementation(deps.okio.fakefilesystem)
-  testImplementation(deps.assertj)
-  testImplementation(deps.kotlin.test.junit)
-  testImplementation(project(":wire-test-utils"))
+kotlin {
+  jvm {
+    withJava()
+  }
+  if (kmpJsEnabled) {
+    js {
+      configure(listOf(compilations.getByName("main"), compilations.getByName("test"))) {
+        tasks.getByName(compileKotlinTaskName) {
+          kotlinOptions {
+            moduleKind = "umd"
+            sourceMap = true
+            metaInfo = true
+          }
+        }
+      }
+      nodejs()
+      // TODO(jwilson): fix Okio for JS to support browser() by polyfilling OS.
```

Comment:
Yeah I think this is done?

Reply:
I'll try later and see if anything breaks